Resources/Prototypes/EstacaoPirata/Entities/Mobs/Customization/cyberlimbs/hesphiastos.yml

```diff
@@ -0,0 +1,129 @@
+- type: marking
+  id: CyberLimbsMarkingHesphiastosHead
+  bodyPart: Head
+  markingCategory: Head
+  speciesRestriction: [IPC] 
+  sprites:
+  - sprite: EstacaoPirata/Mobs/Customization/cyberlimbs/hesphiastos/hesphiastos_monitor.rsi
+    state: head-1
+  - sprite: EstacaoPirata/Mobs/Customization/cyberlimbs/hesphiastos/hesphiastos_monitor.rsi
+    state: head-2
+
+# DeltaV
+- type: marking
+  id: CyberLimbsMarkingHesphiastosHeadAlt
+  bodyPart: Head
+  markingCategory: Head
+  speciesRestriction: [IPC] 
+  sprites:
+  - sprite: DeltaV/Mobs/Customization/cyberlimbs/hesphiastos/hesphiastos_alt1.rsi
+    state: head-1
+  - sprite: DeltaV/Mobs/Customization/cyberlimbs/hesphiastos/hesphiastos_alt1.rsi
+    state: head-2
+  - sprite: DeltaV/Mobs/Customization/cyberlimbs/hesphiastos/hesphiastos_alt1.rsi
+    state: head-3
+
+- type: marking
+  id: CyberLimbsMarkingHesphiastosChest
+  bodyPart: Chest
+  markingCategory: Chest
+  speciesRestriction: [IPC] 
+  sprites:
+  - sprite: EstacaoPirata/Mobs/Customization/cyberlimbs/hesphiastos/hesphiastos_main.rsi
+    state: torso-1
+  - sprite: EstacaoPirata/Mobs/Customization/cyberlimbs/hesphiastos/hesphiastos_main.rsi
+    state: torso-2
+
+- type: marking
+  id: CyberLimbsMarkingHesphiastosLArm
+  bodyPart: LArm
+  markingCategory: Arms
+  speciesRestriction: [IPC, Moth, Dwarf, Human, Arachnid, Felinid, Oni, Vulpkanin, HumanoidFoxes, Reptilian] 
```

Comment:
why can non ipcs use these + should have vox added to ones that the sprites go well with